samples/sid_end_device/include/cli/sbdt_shell.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdint.h>
+#include <zephyr/shell/shell.h>
+#include <sid_bulk_data_transfer_api.h>
+
+#define CMD_SBDT_INIT_DESCRIPTION                                                                  \
+	"Initialize the sidewalk bulk data stack, this can only be done after sid init is done"
+#define CMD_SBDT_INIT_ARG_REQUIRED 1
+#define CMD_SBDT_INIT_ARG_OPTIONAL 0
+
+#define CMD_SBDT_DEINIT_DESCRIPTION                                                                \
+	"This will deinit bulk data transfer stack, will send cancel for any ongoing bulk data transfers"
+#define CMD_SBDT_DEINIT_ARG_REQUIRED 1
+#define CMD_SBDT_DEINIT_ARG_OPTIONAL 0
+
+#define CMD_SBDT_CANCEL_DESCRIPTION                                                                \
+	"<file_id> <reason>\n"                                                                     \
+	"This will send a cancel resp with the given file_id, with the given cancel reason\n"      \
+	"       0x0 - None\n"                                                                      \
+	"       0x1 - Generic\n"                                                                   \
+	"       0x3 - File is too big\n"                                                           \
+	"       0x4 - No space for file on the device\n"                                           \
+	"       0x5 - Device has low battery\n"                                                    \
+	"       0x9 - File verification failed\n"                                                  \
+	"       0xB - File already exists\n"                                                       \
+	"       0xE - Invalid block size"
+
+#define CMD_SBDT_CANCEL_ARG_REQUIRED 3
+#define CMD_SBDT_CANCEL_ARG_OPTIONAL 0
+
+#define CMD_SBDT_CFG_DESCRIPTION                                                                                                                                             \
+	" -p -r -fd <finalize resp delay> -fs <finalize response accept> -tr <transfer reequest accept> -trs <transfer request reject reasons> -br <buffer release delay>\n" \
+	"       - This will allow use to manipulate bulk data transfer, by rejecting or accepting transfers or implying successfull verification of file transfer\n"         \
```

Comment:
```suggestion
	"       - This will allow use to manipulate bulk data transfer, by rejecting or accepting transfers or implying successful verification of file transfer\n"         \
```